pkg/network/protocols/http/telemetry.go

```diff
@@ -27,31 +25,24 @@ type Telemetry struct {
 }
 
 func NewTelemetry() *Telemetry {
-	metricGroup := libtelemetry.NewMetricGroup(
-		"usm.http",
-		libtelemetry.OptExpvar,
-		libtelemetry.OptMonotonic,
-	)
+	metricGroup := libtelemetry.NewMetricGroup("usm.http")
 
-	t := &Telemetry{
-		LastCheck:    atomic.NewInt64(time.Now().Unix()),
-		hits1XX:      metricGroup.NewMetric("hits1xx"),
-		hits2XX:      metricGroup.NewMetric("hits2xx"),
-		hits3XX:      metricGroup.NewMetric("hits3xx"),
-		hits4XX:      metricGroup.NewMetric("hits4xx"),
-		hits5XX:      metricGroup.NewMetric("hits5xx"),
-		aggregations: metricGroup.NewMetric("aggregations"),
+	return &Telemetry{
+		metricGroup: metricGroup,
```

Comment:
wondering what's the use to have both the metric group and the actual metrics inside the telemetry object

Reply:
The main motivation here is to use `MetricGroup.Summary()` which simplifies the logging code below a great deal.

Reply:
I think that is what I missed [here](https://github.com/DataDog/datadog-agent/pull/18151/files#r1261642215)

Reply:
Let's add that for the struct